modules/i18n/src/translations/de-DE/portals/common.ts

```diff
@@ -39,6 +39,10 @@ export const common: CommonNS = {
     "apps": "Anwendungen",
     "assignee": "Beauftragter",
     "assignees": "Beauftragte",
+    "asyncOperationErrorMessage": {
+        "description": "Da ist etwas schiefgelaufen",
+        "message": "Ein unerwarteter Fehler ist aufgetreten. Bitte versuchen Sie es später noch einmal."
+    },
```

Comment:
is the description shorter than the message? looks weird. i thing the msg and description should be swapped